src/Pages/ProductDetail/Components/Photos/Photos.scss

```
@@ -0,0 +1,43 @@
+@import '../../../../Styles/common.scss';
+@import '../../../../Styles/reset.scss';
+.Photos {
+  height: 40rem;
+  display: flex;
+  .leftBox {
+    width: 50%;
+    height: 100%;
+    box-sizing: border-box;
+    background-image: url(https://images.unsplash.com/photo-1552566626-52f8b828add9?ixid=MnwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8&ixlib=rb-1.2.1&auto=format&fit=crop&w=1350&q=80);
+  }
+  .rightBox {
+    width: 50%;
+    height: 100%;
+    display: grid;
+    grid-gap: 5px;
+    grid-template-columns: 1fr 1fr;
+    grid-template-rows: 1fr 1fr;
+    background-color: black;
+    .topLeft {
+      background-image: url(https://images.unsplash.com/photo-1414235077428-338989a2e8c0?ixid=MnwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8&ixlib=rb-1.2.1&auto=format&fit=crop&w=1350&q=80);
+      width: 100%;
+      height: 100%;
+    }
+    .topRight {
+      background-image: url(https://images.unsplash.com/photo-1533777857889-4be7c70b33f7?ixid=MnwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8&ixlib=rb-1.2.1&auto=format&fit=crop&w=1350&q=80);
+      background-size: cover;
+      background-repeat: no-repeat;
+      width: 100%;
+      height: 100%;
+    }
+    .bottomLeft {
+      background-image: url(https://images.unsplash.com/photo-1502301103665-0b95cc738daf?ixid=MnwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8&ixlib=rb-1.2.1&auto=format&fit=crop&w=700&q=80);
+      width: 100%;
+      height: 100%;
+    }
+    .bottomRight {
+      background-image: url(https://images.unsplash.com/photo-1467003909585-2f8a72700288?ixid=MnwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8&ixlib=rb-1.2.1&auto=format&fit=crop&w=634&q=80);
+      width: 100%;
+      height: 100%;
+    }
```

```suggestion
div {
      background-size: cover;
      background-repeat: no-repeat;
    }

    div:first-child {
      background-image: url(https://images.unsplash.com/photo-1414235077428-338989a2e8c0?ixid=MnwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8&ixlib=rb-1.2.1&auto=format&fit=crop&w=1350&q=80);
    }

    div:nth-child(2) {
      background-image: url(https://images.unsplash.com/photo-1533777857889-4be7c70b33f7?ixid=MnwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8&ixlib=rb-1.2.1&auto=format&fit=crop&w=1350&q=80);
    }

    div:nth-child(3) {
      background-image: url(https://images.unsplash.com/photo-1502301103665-0b95cc738daf?ixid=MnwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8&ixlib=rb-1.2.1&auto=format&fit=crop&w=700&q=80);
    }

    div:last-child {
      background-image: url(https://images.unsplash.com/photo-1467003909585-2f8a72700288?ixid=MnwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8&ixlib=rb-1.2.1&auto=format&fit=crop&w=634&q=80);
    }
```

이렇게 한 번 해보세요 :)
공통된 값들은 하나로 통일하고, 다른 부분만 따로 주면 되겠죠?
또한 클래스명을 주지 않고 이런식으로 자식선택자를 사용해서 스타일을 부여할 수도 있습니다.

---

추가로 자식선택자를 사용하면, 각 태그들에 부여된 클래스명은 지워도 되겠죠?

---

반영했습니당